src/test/java/encriptionAlgorithms/EncryptionComparatorTest.java

```diff
@@ -0,0 +1,37 @@
+package encriptionAlgorithms;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+class EncryptionComparatorTest {
+    EncryptionComparator encryptionComparator = new EncryptionComparator();
```

Comment:
Try to sort list of encryption algorithms by the comparator

Reply:
done